core/src/test/java/org/elasticsearch/bwcompat/ClusterStateBackwardsCompatIT.java

```diff
@@ -102,6 +100,7 @@ private NodesInfoResponse clusterNodes() {
 
     private TransportClient newTransportClient() {
         Settings settings = Settings.settingsBuilder().put("client.transport.ignore_cluster_name", true)
+                .put("path.home", PathUtils.get(".").toAbsolutePath())
```

Comment:
Hmm - the transport client doesn't really _use_ `path.home` so it can be anything. I can use that instead.